src/software/ai/hl/stp/play/BUILD

```diff
@@ -119,7 +105,11 @@ cc_library(
 cc_library(
     name = "all_plays",
     deps = [
+<<<<<<< HEAD
```

Comment:
looks like you've got merge conflict markers in here. Needs merge conflict fixing